maketest.sh

```diff
@@ -29,3 +35,29 @@ else
 	$MAKE $@
 fi
 
+# For now (while this code is being prototyped) I only want this to be
+# done while running in jenkins, to avoid a potentially destabilising
+# change if people are running other java processes on their local system
+set -x
+if [ ! -z "$EXECUTOR_NUMBER" ]; then
+  if [ "$OSTYPE" = "cygwin" ]; then
+    # Windows code based on https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1669#issuecomment-727863096
+    echo PROCESSCATCH: Showing java processes - one will be the jenkins agent - I will remove any of these created since $JOBSTARTTIME:
```

Comment:
My preference is to not use first person in echo statements (drop 'I will', 'I am going to', etc) and just say 'Remove any...' and 'Attempt to stop...' etc

Reply:
Awww I quite like the idea that you're having a conversation with a sentient machine ... But I'll change it -)